### DISCUSSION TITLE
Understand lifecycle of branch vs release

### DISCUSSION THREAD

**Discussion by tchiotludo** [micronaut-projects/micronaut-core#4591] [General] [answered]
Hi,

I'm heavily use the Micronaut Validation for my internal usage. 

I've an NullPointerException on micronaut 2.2. 
Looking deeply to understand it's failed here :  

https://github.com/micronaut-projects/micronaut-core/blob/8d8e1b680225ca2077f49283d731c327d5d4f559/validation/src/main/java/io/micronaut/validation/validator/DefaultValidator.java#L1322-L1329

because `propertyValue` == null.

I checked the master branch, https://github.com/micronaut-projects/micronaut-core/blob/e07d7c3343eb7d5d14e377a729adeb887c4cca5d/validation/src/main/java/io/micronaut/validation/validator/DefaultValidator.java#L1322

The method is completely rewrite, and I see a unit test that cover must my failed cases, Nice ! 

Looking at the blame : 
https://github.com/micronaut-projects/micronaut-core/blame/master/validation/src/main/java/io/micronaut/validation/validator/DefaultValidator.java#L1336

@graemerocher fix that 2 years ago, and the modification is not on the any release branch. 

How to handle the inclusion from master to release branch ? 
Is there any ETA on this modification on release ? 
Maybe it's a BC fix, I don't know, just seems weird that a commit 2 years ago was not released.

Thanks

**Comment by graemerocher** [comment, marked as answer]
Could just be a regression from the changes. Your issue has been assigned to a milestone. We currently release patch releases every 1-2 weeks and minor releases every 6 weeks (the next minor release is 2.3) so it won't be long before your fix is included I imagine.

**Reply by tchiotludo** [reply]
Thanks for the prompt reply ! 
You seems to have quickly found the information for the next milestone, I can't find the path to the issue. 
But glad to know it will come quickly :)

**Reply by graemerocher** [reply]
Ah actually I thought this was https://github.com/micronaut-projects/micronaut-core/issues/4590

Could you create another issue to track your NPE?

**Reply by tchiotludo** [reply]
No, it's another one, moving a large codebase from micronaut 1.3.7 to 2.2, so I discover a lot of issues at the same time :smile: 

Done here : https://github.com/micronaut-projects/micronaut-core/issues/4593

Thanks
